pkg/cmd/convert.go

```diff
@@ -17,13 +17,40 @@ package cmd
 import (
 	"strings"
 
+	"crypto/sha256"
+	"crypto/tls"
+	"encoding/base64"
 	v1 "k8s.io/client-go/pkg/api/v1"
+	"net/url"
 )
 
 func isClientConfigKey(key string) bool {
 	return key == "url" || key == "name" || key == "type" || key == "id"
 }
 
+func appendCertificatePinningInfoToService(s *ServiceConfig) error {
+	serviceURL, err := url.Parse(s.URL)
+	if err != nil {
+		return err
+	}
+	if serviceURL.Scheme != "https" {
+		return nil
+	}
+	// TODO: Make the InsecureSkipVerify here configurable. I think there will be times when we don't want to allow auto-pinning to unverified certificates.
+	// TODO: Allow for the Host variable to contain a port. So split it and then if there's a port use that, else use 443.
+	conn, err := tls.Dial("tcp", serviceURL.Host+":443", &tls.Config{
+		InsecureSkipVerify: true,
```

Comment:
I might go for a env var here that could be documented, was originally thinking a flag but not sure it would make sense for such limited usecase

Reply:
Yeah that makes sense, thanks